zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumOnlineRegistry.java

```diff
@@ -55,7 +56,10 @@
   public HeliumOnlineRegistry(String name, String uri, File registryCacheDir) {
     super(name, uri);
     registryCacheDir.mkdirs();
-    this.registryCacheFile = new File(registryCacheDir, name);
+
+    UUID registryCacheFileUuid = UUID.nameUUIDFromBytes(uri.getBytes());
```

Comment:
sounds like `UUID.nameUUIDFromBytes` hashes consistently - ie. given the same uri it will return the same uuid?